batchflow/plotter/plot.py

```diff
@@ -185,6 +209,28 @@ def update(self, data):
         if self.mode == 'loss':
             raise NotImplementedError("Updating layer data is not in supported in 'loss' mode. ")
 
+    def _parse_v_ranges(self, data):
+        """ Parse vmin and vmax values from the `self.config`. """
+        vmin = self.config.get('vmin', np.nanmin(data))
+        vmax = self.config.get('vmax', np.nanmax(data))
+        v_common = self.config.get('v_common', None)
```

Comment:
Do you have any other suggestions for naming of this parameters? I am not entirely sure about `v_common`, but don't have better alternatives:(

`vabs`
`vminmax`